Server/ReasnAPI/ReasnAPI/Services/EventService.cs

```diff
@@ -203,17 +193,17 @@ private void RemoveParameters(Event eventToDelete, int eventId)
         }
     }
 
-    private void RemoveComments(Event eventToDelete)
+    private void RemoveCommentsFromEventCollection(Event eventToDelete)
     {
         context.Comments.RemoveRange(eventToDelete.Comments);
     }
 
-    private void RemoveParticipants(Event eventToDelete)
+    private void RemoveParticipantsFromEventCollection(Event eventToDelete)
     {
         context.Participants.RemoveRange(eventToDelete.Participants);
     }

```

Comment:
As discussed, external services should be usued

Reply:
i will change it after #56